### PR TITLE
Add appleid

### PR DESCRIPTION
**Pull request by AlanJIsaacson**
Hi Simon,

I have added notes on adding the Apple ID for staff. I'm not sure which apps we would like people to tick for iCloud to sync. Can you review and add what you think is a suitable list please? 
I would suggest at the least removing News and Stocks

Thanks 

Alan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Standard Mac setup for the team at Studio 24.
 * [Mac setup for all staff](mac-setup.md)
 * [Telephone setup](telephone.md)
 * [Accessing VPN](vpn.md)
+* [Apple ID](appleid.md)
```
Comment (simonrjones) [inline README.md:12]: to follow other file naming can you rename to: apple-id.md

File: appleid.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Apple ID
+Apple IDs are a requirement for Macbook support. 
+## Installation
+You will receive an invitation via email containing
```
Comment (simonrjones) [inline appleid.md:4]: Can you add a line return before lists, the formatting has gone a bit funky https://github.com/studio24/mac-setup/blob/94b12c0285d7e5f0ebc72cd0519f43421a206452/appleid.md

File: appleid.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Once the Apple ID is setup, add it to your Macbook
+* Open System Preferences
+* Open Internet Accounts
+* Select iCloud
+* Enter your Apple ID, then select Next
+* Enter your password, then select Next
+* Enter the verification code
+* Enter your Mac login password
+* Select the following apps for iCloud to sync Notes
```
Comment (simonrjones) [inline appleid.md:26]: What does this mean? We don't really use Notes across the team so I wouldn't suggest it. Presumably nothing is required so people can enable any iCloud services they wish to use?

File: appleid.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Apple ID
+Apple IDs are a requirement for Macbook support. 
+## Installation
+You will receive an invitation via email containing 
+* Your managed Apple ID eg: user@studio24.net.
+* Your temporary password
+
+
+* Click the link in the email or visit the URL [Apple Business Manager](https://business.apple.com)
```
Comment (simonrjones) [inline appleid.md:9]: May be worth having this not in a bullet and in a paragraph

**Review by simonrjones — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AlanJIsaacson) [reply README.md]: Updated

File: appleid.md (hunk 2 above)
Comment (AlanJIsaacson) [reply appleid.md]: Updated

File: appleid.md (hunk 3 above)
Comment (AlanJIsaacson) [reply appleid.md]: Sorry meant to remove the word Notes. Adjusted the text.

File: appleid.md (hunk 4 above)
Comment (AlanJIsaacson) [reply appleid.md]: Updated

File: mac-setup.md [hunk 5]
```diff
@@ -189,6 +190,12 @@ These commands are carried out within the [Addigy portal](https://prod.addigy.co
 * Setup asset tag
 * Setup Finder & firmware password via 'Script' deployment
 
+This is carried out within the [Apple Business Manager portal](https://business.apple.com/):
+
+* Setup a new Apple ID as user@studio24.net
+* Select 'Create Sgn-in'
```
Comment (simonrjones) [inline mac-setup.md:196]: typo in this sentence (Sng-in)

**Review by simonrjones — CHANGES_REQUESTED** [review]

File: mac-setup.md (hunk 5 above)
Comment (AlanJIsaacson) [reply mac-setup.md]: Corrected now

**Review by simonrjones — APPROVED** [review]

**Pull request merged**